src/main/java/duke/manager/TaskList.java

```diff
@@ -0,0 +1,132 @@
+package duke.manager;
+
+import java.util.ArrayList;
+
+import duke.exception.DukeException;
+import duke.task.Task;
+
+/**
+ * Represents the list of current tasks.
+ */
+public class TaskList {
+    /** The user's current tasks. */
+    private ArrayList<Task> list;
+
+    public TaskList() {
+        this.list = new ArrayList<>();
+    }
+
+    public TaskList(ArrayList<Task> storedTasks) {
+        this.list = storedTasks;
+    }
+
+    /**
+     * Returns the number of current tasks.
+     *
+     * @return The number of current tasks.
+     */
+    public int length() {
```

Comment:
```suggestion
    public int getLength() {
```
Perhaps using a verb as the method name?

Reply:
Thanks! Good spot